Marlin/src/feature/mmu/mmu2.cpp

```diff
@@ -971,10 +974,7 @@ bool MMU2::eject_filament(const uint8_t index, const bool recover) {
 
   LCD_MESSAGE(MSG_MMU2_EJECTING_FILAMENT);
 
-  stepper.enable_extruder();
```

Comment:
> filament still in extruder and damaged by mmu unloading.

@JohnnyTheOne @niccoreyes — This is the only line I could see as being different, apart from the items on line 943 above. One of those blocks is the problem, so it just needs to be patched and reviewed again. The block above (line 943) is the only major change I can see here, while other changes are basic refactors. There are a lot of good suggestions by @sjasonsmith for better failsafes, etc., but we should get this (now old) set of minor improvements merged if they do generally improve the UX. We hope to work closer with Prusa on things like this, but we are all pretty busy atm.